src/l500/ac-trigger.cpp

```diff
@@ -839,17 +887,29 @@ namespace ivcam2 {
         auto hwm = _hwm.lock();
         if (!hwm)
         {
-            AC_LOG(DEBUG ,"Hardware monitor is inaccessible - cannot read temperatures");
+            AC_LOG(ERROR, "Hardware monitor is inaccessible - cannot read temperatures");
             return 0.0;
         }
         // The temperature may depend on streaming?
-        auto res = hwm->send( command{ TEMPERATURES_GET } );
+        std::vector<byte> res;
+
+        try
+        {
+            res = hwm->send(command{ TEMPERATURES_GET });
+        }
+        catch (std::exception const & e)
+        {
+            AC_LOG(ERROR,
+                "Failed to get temperatures; hardware monitor in inaccessible");
+            return 0.0;
+        }
+
         if( res.size() < sizeof( temperatures ) )  // New temperatures may get added by FW...
         {
             AC_LOG( ERROR,
                     "Failed to get temperatures; result size= "
                         << res.size() << "; expecting at least " << sizeof( temperatures ) );
-            return 0;
```

Comment:
Last time I checked, 0 == 0.0... :)

Reply:
Yes but if you declare that you return a double, return a double :)
normally a static analysis tool will give a warning on that

Reply:
I still maintain that 0 has no type... but that's got to be the most annoying static analysis warning of all time!

Reply:
I got used to it.. :)